dask/dataframe/tests/test_groupby.py

```diff
@@ -2491,18 +2491,41 @@ def test_groupby_aggregate_categoricals(grouping, agg):
 
 
 @pytest.mark.xfail(
-    not dask.dataframe.utils.PANDAS_GT_110,
+    not PANDAS_GT_110,
     reason="dropna kwarg not supported in pandas < 1.1.0.",
 )
-@pytest.mark.parametrize("dropna", [False, True])
-def test_groupby_dropna_pandas(dropna):
+@pytest.mark.parametrize(
+    "by,dropna",
+    [
+        ("a", False),
+        ("a", True),
+        pytest.param(
+            ["a", "b"],
+            False,
+            marks=pytest.mark.xfail(
```

Comment:
Do we want to xfail, or catch the warning here, or both? thoughts?

Reply:
IMO, xfail is best here. Once pandas 1.4 is out of circulation, this test will pass without issue we can clean up the parametrization when convenient.